### ISSUE TITLE
Z-Wave Wink siren/chime plays no sound when activated anymore

### ISSUE THREAD

**Issue by weekendwarrior1999** [home-assistant/addons#3165]
### Describe the issue you are experiencing

When activated the siren/chime's indicator light flashes (alarm), but no sound (siren) is emitted (usually is ear piercingly loud). But indicates they are actively communicating with HA. I have two of these and the batteries went dead about a couple of months or so ago, and just got around to ordering new batteries, and now both do the same thing. They had previously worked flawlessly. I even tried factory resetting them and resetting them back up, to which ZwaveJS detected them, but there wasn't any behavior change. Also tried restarting ZwaveJS, and the entire HA machine which also resulted in no change. Obviously, there have been several updates to HA that's been applied during that time, so I'm not exactly sure when the functionality was lost. I also noticed that for the device sensors, the "alarmLevel" and "alarmType" are constantly shown as "0" regardless if activated or not, even though the Alarm Volume level is set to "High Volume" and the Alarm Tone is set. These settings can be changed to anything, and it does not change the sensor's readout.

These are the devices I'm using:

https://www.wink.com/products/wink-siren-and-chime/

  

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Which add-on are you reporting an issue with?

Z-Wave JS

### What is the version of the add-on?

0.1.85

### Steps to reproduce the issue

1. Goto Settings > Devices and Services > Zwave > {num} Devices > Wink Alarm
2. On "Controls" section, switch on "Wink Alarm"
3. Flashing indicator lights on siren/chimes flash while turned on, but no sound is emitted.
...

### System Health information

Some MQTT entities have an entity name that starts with the device name. This is not expected. To avoid a duplicate name the device name prefix is stripped of the entity name as a work-a-round. Please inform the maintainer of the software application that supplies the affected entities to fix this issue.

List of affected entities:

sensor.basement_stairs_primarysw_status
sensor.study_status
sensor.basement_stairs_status
sensor.tasmota_status
sensor.bathroom_lights_status
sensor.diningroomlights_status
sensor.livingroomceilingfan1_status
sensor.upstairs_ac_status
sensor.frontporchlight_status
sensor.kitchen_sink_status
sensor.upstairsheater_status

**Comment by weekendwarrior1999** [comment]
Pretty sure I located the issue. the batteries I replaced in the unit are CR123A's but they are rated for 3.7V, not 3V. Pretty sure I've burnt the siren out of these units. Confirmed this by pairing 2 new devices with HA that came with batteries.... Worked as expected. Put the batteries from the new units in the old unit... No sound... so now I'm gonna see if I can find replacement parts for units in an attempt to salvage them.
